ferrocene/doc/user-manual/src/targets/index.rst

```diff
@@ -102,6 +102,18 @@ should not be used in production.
      - Bare-metal
      - \-
 
+   * - :target:`aarch64-unknown-nto-qnx710`
+     - ``aarch64-unknown-nto-qnx710``
+     - Cross-compilation
+     - Full
+     - Only available on targets supported by QNX (:ref:`x86_64-unknown-linux-gnu` and :ref:`x86_64-pc-windows-msvc`)
```

Comment:
I'd write it like "Can only be cross-compiled from :ref:`x86_64-unknown-linux-gnu` and :ref:`x86_64-pc-windows-msvc`"